test/integration/short/client-execute-tests.js

```diff
@@ -539,6 +539,55 @@ describe('Client', function () {
         done();
       });
     });
+    vdescribe('3.0.16', 'with noCompact', function () {
+      before(function (done) {
+        // C* 4.0 does not support compact storage
+        if (helper.isCassandraGreaterThan('4.0')) {
+          this.skip();
```

Comment:
I think you should exit after skipping.

Reply:
I've looked it up in the doc:

> Best practice: To avoid confusion, do not execute further instructions in a test or hook after calling `this.skip()`.

It probably doesn't work throwing an error as other test frameworks, but I haven't looked it up.

Reply:
ah!  TIL.  I'll add returns where applicable 👍